ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -249,6 +249,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-usgov-ipi-private
+- as: e2e-azure-ipi-kubevirt-hypershift-mgmt-p2
```

Comment:
Only add in 4.11 ? I would expect to add it in 4.12 (or both 4.11 and 4.12).

Reply:
We will add another PR for 4.12. Currently, there is an [nlb bug](https://issues.redhat.com/browse/OCPBUGS-2946) blocking hypershift for OCP 4.12 install & creating on aws. We will investigate it on 4.12 azure & aws toghether for kubevirt hypershift.

Reply:
Ack